src/main/java/io/vertx/ext/consul/impl/ConsulClientImpl.java

```diff
@@ -290,6 +349,224 @@ public Future<TxnResponse> transaction(TxnRequest request) {
+    io.vertx.ext.consul.token.AclToken token
+  ) {
+    Promise<io.vertx.ext.consul.token.AclToken> promise = Promise.promise();
+    updateAclToken(accessorId, token, promise);
+    return promise.future();
+  }
+
+  @Override
+  public ConsulClient cloneAclToken(
+    String accessorId,
+    CloneAclTokenOptions cloneAclToken,
+    Handler<AsyncResult<io.vertx.ext.consul.token.AclToken>> resultHandler
+  ) {
+    requestObject(HttpMethod.PUT, "/v1/acl/token/" + urlEncode(accessorId) + "/clone", null,
+      cloneAclToken.toJson().encode(),
+      resultHandler,
+      (obj, headers) -> new io.vertx.ext.consul.token.AclToken(obj)
+    );
+    return this;
+  }
+
+  @Override
+  public Future<io.vertx.ext.consul.token.AclToken> cloneAclToken(
+    String accessorId,
+    CloneAclTokenOptions cloneAclToken
+  ) {
+    Promise<io.vertx.ext.consul.token.AclToken> promise = Promise.promise();
+    cloneAclToken(accessorId, cloneAclToken, promise);
+    return promise.future();
+  }
+
+  @Override
+  public ConsulClient getAclTokens(Handler<AsyncResult<List<io.vertx.ext.consul.token.AclToken>>> resultHandler) {
+    requestArray(HttpMethod.GET, "/v1/acl/tokens", null, null, resultHandler, (arr, headers) ->
+      arr.stream()
+        .map(obj -> new io.vertx.ext.consul.token.AclToken((JsonObject) obj))
+        .collect(Collectors.toList()));
+    return this;
+  }
+
+  @Override
+  public Future<List<io.vertx.ext.consul.token.AclToken>> getAclTokens() {
+    Promise<List<io.vertx.ext.consul.token.AclToken>> promise = Promise.promise();
+    getAclTokens(promise);
+    return promise.future();
+  }
+
+  @Override
+  public ConsulClient readAclToken(
+    String accessorId,
+    Handler<AsyncResult<io.vertx.ext.consul.token.AclToken>> resultHandler
+  ) {
+    requestObject(HttpMethod.GET, "/v1/acl/token/" + urlEncode(accessorId), null, null,
+      resultHandler,
+      (obj, headers) -> new io.vertx.ext.consul.token.AclToken(obj)
+    );
+    return this;
+  }
+
+  @Override
+  public Future<io.vertx.ext.consul.token.AclToken> readAclToken(String accessorId) {
+    Promise<io.vertx.ext.consul.token.AclToken> promise = Promise.promise();
+    readAclToken(accessorId, promise);
+    return promise.future();
+  }
+
+  @Override
+  public ConsulClient deleteAclToken(String accessorId, Handler<AsyncResult<Boolean>> resultHandler) {
+    requestString(HttpMethod.DELETE, "/v1/acl/token/" + urlEncode(accessorId), null, null,
+      resultHandler,
+      (str, headers) -> Boolean.parseBoolean(str)
+    );
+    return this;
+  }
+
+  @Override
+  public Future<Boolean> deleteAclToken(String accessorId) {
+    Promise<Boolean> promise = Promise.promise();
+    deleteAclToken(accessorId, promise);
+    return promise.future();
+  }
+
   @Override
   public ConsulClient createAclToken(AclToken token, Handler<AsyncResult<String>> idHandler) {
     requestObject(HttpMethod.PUT, "/v1/acl/create", null, token.toJson().encode(), idHandler, (obj, headers) ->
```

Comment:
I think this endpoint also needs to be modified to `/v1/acl/token` to match what's added on line 459 above.

Reply:
Ah, I see. I missed the deprecation in the interface, but it makes sense to keep the old mechanism around so that one can still interface with old servers. Thanks!